dhall-mode.el

```diff
@@ -159,8 +151,8 @@ Should be dhall or the complete path to your dhall executable,
           (write-region nil nil bufferfile)
           (with-current-buffer errbuf
             (erase-buffer))
-          (apply 'call-process dhall-format-command nil errbuf t (append dhall-format-options (list
-                                                                                               (buffer-file-name))))
+          (apply 'call-process dhall-command nil errbuf t (append dhall-format-options (list
+                                                                                       (buffer-file-name))))
```

Comment:
I feel like this should explicitly `(cons "format" dhall-format-options)` rather than adding `"format"` to the customizable options.